README.md

```diff
@@ -12,26 +21,59 @@ The difference is that `nv` _feeds_ an explicit environment to the process it st
 
 `nv` is also not language-specific nor framework-specific — it just _feeds_ some environment into the command it’s given to run.
 
-## Install
 
-You can install `nv` as a [Homebrew](https://brew.sh) package on macOS:
+## Installation
 
-```
-~> brew install jcouture/nv/nv
-```
+### macOS
+
+`nv` is available via [Homebrew](#homebrew) and as a downloadable binary from the [releases page](https://github.com/jcouture/nv/releases).
+
+#### Homebrew
+
+| Install                       | Upgrade           |
+|-------------------------------|-------------------|
+| `brew install jcouture/nv/nv` | `brew upgrade nv` |
+
+### Linux
 
-Or you can build it from source:
+`nv` is available as downloadable binaries from the [releases page](https://github.com/jcouture/nv/releases).
 
+### Windows
+
+`nv` is available as a downloadable binary from the [releases page](https://github.com/jcouture/nv/releases).
+
+### Build from source
+
+Alternatively, you can build it from source.
+
+1. Verify you have Go 1.16+ installed
+
+```sh
+~> go version
 ```
-~> mkdir nv
+If `Go` is not installed, follow the instructions on the [Go website](https://golang.org/doc/install)
+
+2. Clone this repository
+
+```sh
+~> git clone https://github.com/jcouture/nv.git
 ~> cd nv
-~> set -x GOPATH $PWD
-~> go get -u github.com/jcouture/nv
 ```
 
+3. Build
+
+```sh
+~> go mod tidy
+~> go build ./cmd/nv/
+```
+
+While the development version is a good way to take a peek at `nv`’s latest features before they get released, be aware that it may contains bugs. Officially released versions will generally be more stable.
+
+
+
```

Comment:
If you want, you could run [Prettier](https://prettier.io/) on the file to ensure consistent spacing and formatting 🤓

Reply:
Great idea, thanks!